weather-app/src/components/DayForecast.js

```diff
@@ -0,0 +1,64 @@
+import Component from '../blackbox';
+import { drawIcon } from '../utils/icons';
+
+class DayForecast extends Component {
+  constructor() {
+    super();
+  }
+
+  getWeekday(datetime) {
+    let weekday = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+    let date = new Date(datetime);
+    return weekday[date.getDay()];
+  }
+
+  tempConverter(temp, units = this.props.units) {
+    if (units === 'FA') return Math.round((temp * 1.8) + 32);
+    else return Math.round(temp);
+  }
+
+  render() {
+    const { forecast, day } = this.props;
+
+    if (forecast.data) {
+      const city = forecast.city_name;
+      const country = forecast.country_code;
+      const weekday = this.getWeekday(forecast.data[day].datetime);
```

Comment:
Can you move logic of getting data from `forecast.data[day]` to some separate util?

Or use decomposition, like: 
```js
const { datetime, wind_spd: wind, rh: humidity, temp, min_temp, max_temp } = forecast.data[day];
```

I think this will make code cleaner.